setup.py

```diff
@@ -1,8 +1,10 @@
-from setuptools import setup  # Always prefer setuptools over distutils
+import sys
 from codecs import open  # To use a consistent encoding
 from os import path
 import re
 
+from setuptools import setup  # Always prefer setuptools over distutils
+
```

Comment:
A detail: avoid committing unrelated changes (moving this `import` line has nothing to do with the conditional inclusion of pytest-runner).

Reply:
I moved it to import `sys`